app/javascript/components/student/editor/__mocks__/ExercismMonacoEditor.tsx

```diff
@@ -1,5 +1,11 @@
 import React, { useEffect, useRef } from 'react'
 
+export enum Keybindings {
```

Comment:
Is there any other way to do this? I've mocked this module since jest doesn't work well with Monaco, but I need this enum defined as well.

Is it a good idea instead to declare a global enums file or something like that?

Reply:
(@SleeplessByte)

Reply:
What is "this"?

I can't quite figure out what you need, or what doesn't work.

Reply:
I've mocked this component out in our tests, as how Monaco imports modules doesn't work well with Jest yet. The only problem is that within this component, I define this `Keybindings` enum. I end up with two versions of the enum which is in the actual component, and within the mocked component.

Do we have a way of defining a global enum or something like that? Or is it okay to do it this way still?

Reply:
I'd probably declare it in a different file (module) and import it in both places, but I might be missing why that doesn't work.